stock_cycle_count/reports/report_stock_location_accuracy.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class LocationAccuracyReport(models.AbstractModel):
+    _name = "report.stock_location_accuracy"
+
+    @api.model
+    def _get_inventory_domain(self, loc_id):
+        return [('location_id', '=', loc_id),
+                ('exclude_sublocation', '=', True),
```

Comment:
Is this shouldn't be optional through parameter ?

Reply:
yes, why not?